### PR TITLE
Add multiattach field to CloudVolume

### PR DESCRIPTION
**Pull request by mansam**
Adds a "multiattach" boolean column to the CloudVolume model
https://bugzilla.redhat.com/show_bug.cgi?id=1660896

### CODE REVIEW

File: db/migrate/20190110201414_add_multiattach_to_cloud_volume.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddMultiattachToCloudVolume < ActiveRecord::Migration[5.0]
+  def change
+    add_column :cloud_volumes, :multiattach, :boolean
```
Comment (agrare) [inline db/migrate/20190110201414_add_multiattach_to_cloud_volume.rb]: Hm `multiattach` looks overly shortened when it doesn't need to be, how about `multi_attach` or `multiple_attachment`?

Comment (aufi) [reply db/migrate/20190110201414_add_multiattach_to_cloud_volume.rb]: Correct, but it matches to OpenStack keywords - https://docs.openstack.org/cinder/latest/admin/blockstorage-volume-multiattach.html

Comment (agrare) [reply db/migrate/20190110201414_add_multiattach_to_cloud_volume.rb]: True but this is provider common code so matching any specific provider isn't as important as it matching the rest of the platform consistently.

Comment (mansam) [reply db/migrate/20190110201414_add_multiattach_to_cloud_volume.rb]: @agrare multi_attach or multiple_attachment would suit me fine, do you have a preference?

Comment (agrare) [reply db/migrate/20190110201414_add_multiattach_to_cloud_volume.rb]: @mansam I think `multi_attachment` would be good, I'm bad at naming :)

**Pull request merged**